### DISCUSSION TITLE
Reference Strip

### DISCUSSION THREAD

**Discussion by HemboPagi** [openseadragon/openseadragon#2182] [Q&A]
Hello

Can someone point me to the examples were custom reference strip is working.
I would like to have it in my own div as well as remove mouseover animations.

Many thanks in advance

Hembo

**Comment by iangilman** [comment]
As discussed in https://github.com/openseadragon/openseadragon/issues/333 it seems that the custom reference strip is broken. If you'd like to help fix it, that would be lovely! 

Another approach would be to build your own reference strip just like you like it, using the built-in reference strip's code as a starting point: 

https://github.com/openseadragon/openseadragon/blob/master/src/referencestrip.js
